### ISSUE TITLE
Stale errors (both spans and messages)

### ISSUE THREAD

**Issue by JsonFreeman** [Microsoft/TypeScript-Sublime-Plugin#50]
Type the following:

``` ts
C
```

As expected, you get an error on C: "Cannot find name C".

Now insert the word "static" before "C".

Expected: New error "Declaration or statement expected" on the word "static".
Actual: Status bar still shows old message with old span.
